pkg/templates/kubeadm/v1beta3/kubeadm.go

```diff
@@ -590,6 +590,14 @@ func newNodeRegistration(s *state.State, host kubeoneapi.HostConfig) kubeadmv1be
 		kubeletCLIFlags["max-pods"] = strconv.Itoa(int(*m))
 	}
 
+	if m := host.Kubelet.PodPidsLimit; m != nil {
+		kubeletCLIFlags["pod-max-pids"] = strconv.Itoa(int(*m))
+	} else {
+		// Set default value if PodsPidsLimits is nil
+		// in order to pass the 4.2.13 Check in CIS Benchmark 1.8
+		kubeletCLIFlags["pod-max-pids"] = strconv.Itoa(-1)
+	}
```

Comment:
Have we checked if this solves the CIS benchmark issue or they require setting a value that's not -1?

Reply:
CIS Benchmark requires to put a value and don't rely on kubelet default value. So in our case if the pod pid limit we will put the default value of kubelet to pass the test

Reply:
That's great, thanks for following up on this!